src/components/RepCard/RepCard.css

```diff
@@ -0,0 +1,22 @@
+.rep-image {
+  width: 200px;
+  float: left;
+  box-align: left;
```

Comment:
Looks like `box-align` isn't supported by most browsers so we should remove it
<img width="532" alt="Screen Shot 2021-10-08 at 3 01 48 PM" src="https://user-images.githubusercontent.com/22085207/136618594-dec3e4c4-31cb-4b6f-9fd2-4e39a5909b77.png">